src/components/FlippableHandler.js

```diff
@@ -30,7 +30,9 @@ class FlippableHandler {
                         window.setTimeout(() => $newEl.classList.remove('display-flex'), 600);
                         $oldEl.classList.add('display-flex');
                         window.setTimeout(() => $oldEl.classList.remove('display-flex'), 300);
-                        $rotationContainer.classList.toggle('flipped', $newEl.classList.contains(classname));
+                        window.setTimeout(() => {
+                            $rotationContainer.classList.toggle('flipped', $newEl.classList.contains(classname));
+                        }, 0);
```

Comment:
As always: Comments are great.
And this could be written in one line, i.e. without opening a `{ }` block.

Reply:
It does not fit in one line if I strip the `{ }`. I choose to have it in this form as it reads better in my opinion. 
As all the toggles have the same setTimeout I don't think a comment was warranted.